src/_posts/security/bulletins/2024-03-13-SSB-2024-002.md

```diff
@@ -0,0 +1,106 @@
+> 
+> 
+> This is the most secure of the currently provided methods, but it is not supported by older client libraries.
+
+This is a per-database-user change. It implies resetting the users' passwords (since PostgreSQL doesn't know them).
+
+## Timeline of the rollout
+
+There are 2 distinct timelines, depending on the type of the parent application of the PostgreSQL add-on.
+
+- **Health Data Hosting databases (HDH):** for PostgreSQL add-ons linked to HDH applications, this update is scheduled for $MIGRATION_START_DATE+6_MONTHS (in 6 months' time). This notice is due to the nature of the data contained in these databases and the excessive caution they require.
+- **Other databases:** this change will be applied automatically **in the coming weeks**.
+
+You will be notified by e-mail 24 hours before the start of the maintenance period of the affected add-on(s), according to the database maintenance window configuration.
+
+## Impact on your applications
+
+Rest assured, this transition is designed to have minimal impact on your applications (less than a second). The SCRAM-SHA-256 algorithm is widely supported by PostgreSQL clients library and has been in use for several years.
+
+To ensure a smooth transition, we kindly request you to verify that your PostgreSQL client version supports the SCRAM-SHA-256 algorithm. You can refer to the [PostgreSQL Wiki](https://wiki.postgresql.org/wiki/List_of_drivers) for the list of compatible client library versions.
+
+## Application of this change
+
+As PostgreSQL stores only a *hash* of the password, changing the hashing algorithm implies that we provide the original password again to the database. This is only possible for users that are managed by Scalingo (the *protected* users in your database dashboard).
+
+### Protected users
+
+Protected users are credentials created automatically by Scalingo to allow applications to connect to databases.
+
+![Database users panel with a protected user](https://cdn.scalingo.com/documentation/security/20240313_database_users_panel_protected_user.png)
+
+As we know the passwords for these users, we will be able to update the passwords automatically.
+
+All protected users in your PostgreSQL databases hosted on Scalingo will automatically be migrated to use SCRAM authentication.
+
+### Unmanaged users
+
+Unmanaged users of your database **won't be migrated** as we don't know their passwords.
+
+
+{% note %}
+  Not migrating these users will not affect your database operations, but it will reduce your level of security.
+{% endnote %}
+
+If you have created users, either via the CLI or via the database dashboard is the same.
+
+Once you have been notified that your database has been migrated. 
```

Comment:
What do you want to say?

Reply:
Same here, that's what you wrote on January 16th. Would you like to rephrase?